scripts/setup/security.ps1

```diff
@@ -206,17 +224,18 @@ function AddUserToGroup($userPath, $_group) {
         catch {
             # HRESULT -2147023518
             # The specified account name is already a member of the group.
-            if($_.Exception.InnerException -eq $null -or ($_.Exception.InnerException.HResult -ne -2147023518 -and $_.Exception.InnerException.ErrorCode -ne -2147023518)) {
+            if($_.Exception.InnerException -or ($_.Exception.InnerException.HResult -ne -2147023518 -and $_.Exception.InnerException.ErrorCode -ne -2147023518)) {
                 throw $_.Exception
             }
         }
     }
     else {
-        $existingMember = Get-LocalGroupMember -Group $_group.name | where {$_.Name -eq $userPath}
-
-        if ($existingMember -eq $null) {
-            Add-LocalGroupMember -Name $($_group.name) -Member $($userPath)
-        }
+        $existingMember = Get-LocalGroupMember -Group $groupName | Where {$_.Name -eq $userPath}
+        if (!$existingMember) {
+            Add-LocalGroupMember -Name $($groupName) -Member $($userPath)
```

Comment:
Please put trace log like "Write-Verbose "Trying to Add-LocalGroupMember"" before 235 line executes.